apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.js

```diff
@@ -110,26 +110,22 @@
 
     let inputElement = $('#rte-input-' + uuid);
     let quill = new Quill('#rte-' + uuid, options);
-    let renderer = null;
-    if (options.format === 'markdown') {
-      renderer = new showdown.Converter({
-        literalMidWordUnderscores: true,
-        literalMidWordAsterisks: true,
-      });
-    }
 
     if (options.markdownShortcuts && !options.readOnly) new QuillMarkdown(quill, {});
 
     let contents = atob(inputElement.val());
-    if (contents && renderer) contents = renderer.makeHtml(contents);
+    if (contents && options.format === 'markdown') {
+      const marked = (await import('marked')).marked;
```

Comment:
Since pl-rich-text-editor has some different ways to handle math, and assuming it's not expected to be available in the provided text from the question, I haven't added support for mathjax here.